trio/lowlevel.py

```diff
@@ -7,10 +7,12 @@
 import sys
 import typing as _t
 
+# Generally available symbols
 # Generally available symbols
```

Comment:
Duplicated comment?

Reply:
`isort` probably merged two sets of imports from `_core`